database/dtb.ksy

```diff
@@ -0,0 +1,124 @@
+meta:
+  id: dtb
+  title: Flattened Devicetree Format
+  file-extension: dtb
```

Comment:
```yaml
application:
  - Das U-Boot
  - Linux
```

Reply:
added, there are probably more that use it. We can add those later.